docs/pages/errors/index.vue

```diff
@@ -0,0 +1,218 @@
+        User errors
+      </h3>
+      <p>
+        User errors occur when a user input or interaction isn’t understood by the system. Examples include invalid field inputs and accessing blocked pages.
+      </p>
+      <h3>
+        System errors
+      </h3>
+      <p>
+        System errors are usually caused by factors that are not the fault of the user. Examples include connectivity or network issues, missing data or page not found, and other unexpected issues.
+      </p>
+    </DocsPageSection>
+
+    <DocsPageSection title="Language" anchor="#language">
+      <p>
+        Follow the <DocsInternalLink href="/writing">
+          writing guidelines
+        </DocsInternalLink> with a special focus on clear, concise, and natural language. Explain causes of errors if it will help the user resolve their issue, but do not use complex technical terms and run-on explanations.
+      </p>
+      <p>
+        Error messages should also be actionable. Offer the solution as an action when possible and make sure the action label is specific to the outcome. If including an action isn’t possible, then briefly explain how the user can fix their problem.
+      </p>
+      <DocsDoNot>
+        <template v-slot:do>
+          <p class="do-dont">
+            “Sorry, something went wrong. <u>Refresh page</u>"
+          </p>
+        </template>
+        <template v-slot:not>
+          <p class="do-dont">
+            “500 internal server error. It looks like the server failed to fulfill an apparently valid GET or POST request. We are aware of this issue and you can try solving it by reloading the page. We apologize for the inconvenience and hope this doesn’t occur again.”
+          </p>
+        </template>
+      </DocsDoNot>
+      <h3>
+        Tone
+      </h3>
+      <p>
+        Let the user know what went wrong without blaming them. Do not use all caps, exclamation points, or other alarmist language.
+      </p>
+      <DocsDoNot>
+        <template v-slot:do>
+          <p class="do-dont">
+            “Please enter a valid email”
+          </p>
+        </template>
+        <template v-slot:not>
+          <p class="do-dont">
+            “WARNING!! You didn’t fill this out”
+          </p>
+        </template>
+      </DocsDoNot>
+      <p>
+        Even though errors can induce negative emotions, avoid being too whimsical and lighthearted. This can have the opposite effect and cause more frustration.
+      </p>
+      <DocsDoNot>
+        <template v-slot:do>
+          <p class="do-dont">
+            “Storage limit reached”
+          </p>
+        </template>
+        <template v-slot:not>
+          <p class="do-dont">
+            “Whoops! Looks like the tank is too full &#x1f60e;"
+          </p>
+        </template>
+      </DocsDoNot>
+    </DocsPageSection>
+
+    <DocsPageSection title="Components and patterns" anchor="#components">
+      <h3>
+        Color
+      </h3>
+      <DocsColorBlock name="tokens.error" definition />
```

Comment:
In this case, we should remove the `definition` property, because including it creates an anchor link which does not work outside the Colors page.

I opened a follow-up issue here: https://github.com/learningequality/kolibri-design-system/issues/98

```suggestion
      <DocsColorBlock name="tokens.error" />
```